azurepipelines/e2e_test/e2etest.yaml

```diff
@@ -128,7 +128,7 @@ stages:
                       package_pipeline: "Azure.adu-private-preview.e2e-test"
                       scenarioSetupDir: "./scenarios/ubuntu-18.04-amd64"
                       du_tarball_script: >-
-                          tar -xvf /tmp/testsetup.tar.gz -C ./ &&
+                          tar -xf /tmp/testsetup.tar.gz -C ./ &&
```

Comment:
curious why verbose ("v" - file list) was removed. was it interfering with something else?

Reply:
(throughout PR)